Podcast/ExternalProfileViewController.swift

```diff
@@ -47,58 +44,68 @@ class ExternalProfileViewController: ViewController, UITableViewDataSource, UITa
         profileHeaderEmptyFrame.backgroundColor = .sea
         view.addSubview(profileHeaderEmptyFrame)
         
-        backButton = UIButton(type: .custom)
-        backButton.frame = CGRect(x: padding, y: ProfileHeaderView.statusBarHeight + (ProfileHeaderView.miniBarHeight - ProfileHeaderView.statusBarHeight - backButtonHeight) / 2, width: backButtonWidth, height: backButtonHeight)
-        backButton.setImage(UIImage(named: "backArrowLeft"), for: .normal)
-        backButton.adjustsImageWhenHighlighted = true
-        backButton.addTarget(self, action: #selector(didPressBackButton), for: .touchUpInside)
-        view.addSubview(backButton)
-        view.bringSubview(toFront: backButton)
-        
-        loadingAnimation = createLoadingAnimationView()
-        loadingAnimation.center = view.center
-        view.addSubview(loadingAnimation)
-        loadingAnimation.startAnimating()
-        UIApplication.shared.statusBarStyle = .default
-    }
-    
-    func createSubviews() {
-        
-        // Instantiate tableView
-        profileTableView = UITableView(frame: CGRect(x: 0, y: 0, width: view.frame.width, height: view.frame.height), style: .grouped)
+        profileTableView = UITableView(frame: .zero, style: .grouped)
         for (contentClass, identifier) in zip(sectionContentClasses, sectionContentIndentifiers) {
             profileTableView.register(contentClass.self, forCellReuseIdentifier: identifier)
         }
         profileTableView.delegate = self
         profileTableView.dataSource = self
         profileTableView.backgroundColor = .paleGrey
+        profileTableView.rowHeight = UITableViewAutomaticDimension
         profileTableView.separatorStyle = .none
         profileTableView.showsVerticalScrollIndicator = false
         mainScrollView = profileTableView
         view.addSubview(profileTableView)
+        profileTableView.snp.makeConstraints { make in
+            make.edges.equalToSuperview()
+        }
         
-        let backgroundExtender = UIView(frame: CGRect(x: 0, y: 0-profileTableView.frame.height+20, width: profileTableView.frame.width, height: profileTableView.frame.height))
+        let backgroundExtender = UIView(frame: CGRect(x: 0, y: 0-view.frame.height+20, width: view.frame.width, height: view.frame.height))
```

Comment:
^^ I think it's more optimized if it's 0- instead of -1*

Reply:
It's so when you pull down on the profile, you see more green instead of the paleGrey. But the background has to be paleGrey for the bottom when you scroll past.